src/start_server.ts

```diff
@@ -456,6 +468,23 @@ async function startWithFirstAvailablePort(
 async function tryStartWithPort(
     options: ServerOptions, app: express.Application, port: number) {
   const server = await createServer(app, options);
+
+  if (options.liveReloadPath) {
+    const wss = new WebSocketServer({server: server});
+    wss.on('connection', (ws: WebSocket) => {
+      const watcher = watchFile(options.liveReloadPath);
```

Comment:
The naming was confusing as I aliased `watch` to `watchFile` which is actually incorrect. I changed this back to just `watch`, as passing directories/files/globs etc... will all be handled by `chokidar`. E.g. when passing a directory such as `src/`, chokidar will listen for sub-directories.